boards.txt

```diff
@@ -339,28 +347,29 @@ CH32V10x_EVT.menu.upload_method.swdMethod.upload.protocol=
 CH32V10x_EVT.menu.upload_method.swdMethod.upload.options=
 CH32V10x_EVT.menu.upload_method.swdMethod.upload.tool=WCH_linkE
 
-# Clock Select 
+
+# Clock Select
 CH32V10x_EVT.menu.clock.72MHz_HSI=72MHz Internal
 CH32V10x_EVT.menu.clock.72MHz_HSI.build.flags.clock=-DSYSCLK_FREQ_72MHz_HSI=72000000
 CH32V10x_EVT.menu.clock.56MHz_HSI=56MHz Internal
 CH32V10x_EVT.menu.clock.56MHz_HSI.build.flags.clock=-DSYSCLK_FREQ_56MHz_HSI=56000000
 CH32V10x_EVT.menu.clock.48MHz_HSI=48MHz Internal
 CH32V10x_EVT.menu.clock.48MHz_HSI.build.flags.clock=-DSYSCLK_FREQ_48MHz_HSI=48000000
 CH32V10x_EVT.menu.clock.8MHz_HSI=8MHz Internal
-CH32V10x_EVT.menu.clock.8MHz_HSI.build.flags.clock=-DSYSCLK_FREQ_HSI=8000000
+CH32V10x_EVT.menu.clock.8MHz_HSI.build.flags.clock=-DSYSCLK_FREQ_8MHz_HSI=8000000
```

Comment:
is this typo? should it be -DSYSCLK_FREQ_HSI/HSE=8000000 or -DSYSCLK_FREQ_8MHz_HSI/HSE=8000000 to be consistent with above defines